ballerina/tests/mock_tests.bal

```diff
@@ -0,0 +1,243 @@
+// Copyright (c) 2024 WSO2 LLC. (http://www.wso2.org).
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+import ballerina/log;
+
+listener http:Listener httpListener = new (9090);
+
+http:Service mockAPI = service object {
+
+    # Get Presence
+    #
+    # + return - Get Presence
+    resource isolated function get users\.getPresence() returns json => {
+        response: {
+            "ok": true,
+            "presence": "away",
+            "online": false,
+            "auto_away": false,
+            "manual_away": false,
+            "connection_count": 0
+        }
```

Comment:
It seems the correct response structure should be
```suggestion
            "ok": true,
            "presence": "away",
            "online": false,
            "auto_away": false,
            "manual_away": false,
            "connection_count": 0
```

Need to update all the resource functions accordingly